### PR TITLE
Introduce AND/OR and conditional input

### PR DESCRIPTION
**Pull request by MrGeneration**

### CODE REVIEW

**Comment by MrGeneration** [comment]
@Suhopoljac Please check for logical flaws and if the new area within Misc leaves questions open or even raises more than it answers.

@dvuckovic Please check `misc/object-conditions/expert-mode.rst` if the technical remarks are correct and logically correct. 🙏

File: misc/object-conditions/time-conditions-mermaid.include.rst [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+      section after<br>(relative)
+      2 days       :after withinnext, 2d
+      section within last<br>(relative)
+      2 days       :withinlast, after beforeabs, 2d
+      section within next<br>(relative)
+      2 days       :withinnext, after withinlast, 2d
+      section til<br>(relative)
+      14.06.       :09.06.2023, 5d
+      section from<br>(relative)
+      11.06.       :after beforeabs, 5d
```
Comment (dvuckovic) [inline misc/object-conditions/time-conditions-mermaid.include.rst:24]: Screenshot 2023-02-07 at 09 58 49

File: manage/trigger/limitations.rst [hunk 2]
```diff
@@ -10,6 +10,11 @@ Triggers will fire during the following conditions:
   * Creation of a ticket
   * Updating a ticket
 
+  .. important::
+
+    Triggers are only accounted if you're adding a article to the ticket.
```
Comment (dvuckovic) [inline manage/trigger/limitations.rst:15]: _adding **a** article_ => _adding **an** article_

**Review by dvuckovic — APPROVED** [review]
@MrGeneration Looks pretty good already, I have only a minor wording suggestions. And possibly graph improvements, at your discretion. Thanks!

File: misc/object-conditions/expert-mode.rst [hunk 3]
```diff
@@ -0,0 +1,204 @@
...
+   MTLB ==> MSLC
+   SLA1 --> MSLC
+   SLA2 --> MSLC
+   SLB1 --> MSLD
+   SLB2 --> MSLD
+   TLA  --> MFLA
+   TLB  --> MFLA
+   MSLC ==> MFLA
+   MSLD ==> MFLA
+   MFLA ==> Finish
```
[the remark this answers is not in the document]
Comment (MrGeneration) [reply misc/object-conditions/expert-mode.rst:204]: Good point!

File: misc/object-conditions/expert-mode.rst [hunk 4]
```diff
@@ -0,0 +1,204 @@
...
+See `Evaluation order`_ to understand how Zammad evaluates triggers in expert
+mode.
+
+Adding conditions and logic blocks
+----------------------------------
+
+By using the |subclause| icon you can add as many logic blocks as you need.
+New blocks will be added below the object you're using the |subclause| icon on.
+
+The level automatically will be set to a lower level than the object you're
```
Comment (Suhopoljac) [inline misc/object-conditions/expert-mode.rst:66]: The level automatically will be set  --> The level will be automatically set

**Pull request merged**